tests/cleanup/vmware_list_vms.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
```

Comment:
I will merge as-is but this file needs to go back into pyvmomi-community-samples (we can add it to the current branch that we use for composer_testing) until upstream decides to merge it. I will open another task for that.